cmd/util.go

```diff
@@ -34,16 +34,35 @@ func toJSON(dev *ttnpb.EndDevice) ([]byte, error) {
 var (
 	errExport = errors.Define("export", "export device `{device}`")
 	errFormat = errors.DefineCorruption("format", "format device `{device}`")
+
+	errFailedValidation = errors.DefineInvalidArgument("failed_validation", "failed to validate device")
 )
 
+func replaceUnderscores(s string) string {
```

Comment:
Replace with what?

Please be clear and not require people to look at the implementation, even if it's a one liner, to see what's happening.

You can also do a package-global var like:

```go
var sanitizeID = strings.NewReplacer("-", "_")
```

This allows more advanced implementations without touching callers, when we implement some custom `Replace()` method